saleor/webhook/payloads.py

```diff
@@ -155,6 +158,16 @@ def generate_product_payload(product: "Product"):
     return product_payload
 
 
+def _get_sample_object(qs: QuerySet) -> Optional[Model]:
+    """Return random object from query."""
+    count = qs.count()
```

Comment:
Why not the first one every time? Doing a count then getting the `x`th one has a heavy cost

Reply:
What do you mean by saying heavy cost? 
I need to have different results for the sample.

Reply:
What about `order_by("?").first()` then?

Reply:
I found some old tests which point that order_by is heavier than count but I've checked that and it seems that is not relevant anymore. I will change this. Thanks